k8s/pkg/apis/longhorn/v1beta1/backuptarget.go

```diff
@@ -18,9 +18,11 @@ type BackupTargetSpec struct {
 	CredentialSecret string `json:"credentialSecret"`
 	// The interval that the cluster needs to run sync with the backup target.
 	// +optional
+	// +nullable
```

Comment:
This one seems not to be necessary?

Reply:
I think this one does not need to add `// +nullable`.

Reply:
Yes, should remove it.